widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -1638,9 +1638,10 @@ internal class ChatController(
                 dialogController.showEngagementEndedDialog()
             }
 
-            shouldHandleEndedEngagement -> {
+            else -> {
                 // Close chat screen
                 Dependencies.getControllerFactory().destroyControllers()
+                viewCallback?.destroyView()
```

Comment:
It's better to use a dedicated function `destroyView()` I think.

Reply:
Changed